common/tests/functional/Test-throughput.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+#Francis 
+if ! [ -x "$(type -P iperf3)" ]; then
+  echo "ERROR: script requires iperf3"
+  echo "For Debian and friends get it with 'apt-get install iperf3'"
+  echo "If you have it, perhaps you don't have permissions to run it, try 'sudo $(basename $0)'"
+  exit 1
+fi
+
+if [ "$#" -ne "2" ]; then
+  echo "ERROR: script needs four arguments, where:"
+  echo
+  echo "1. Number of times to repeat test (e.g. 10)"
+  echo "2. Host running 'iperf -s' (e.g. somehost)"
```

Comment:
Might be worth to change iperf to iperf3 as I think typing iperf is likely to give "command not found" even if you have iperf3 installed